src/components/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Todo } from './TodoItem';
+
+interface FooterProps {
+  todos: Todo[];
+  filter: string;
+  onFilterChange: (filter: string) => void;
+  onClearCompleted: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  filter,
+  onFilterChange,
+  onClearCompleted,
+}) => {
+  const activeTodosCount = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosCount} items left
+      </span>
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={`filter__link ${filter === 'all' ? 'selected' : ''}`}
+          onClick={() => onFilterChange('all')}
+        >
+          All
+        </a>
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={`filter__link ${filter === 'active' ? 'selected' : ''}`}
+          onClick={() => onFilterChange('active')}
+        >
+          Active
+        </a>
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={`filter__link ${filter === 'completed' ? 'selected' : ''}`}
+          onClick={() => onFilterChange('completed')}
+        >
+          Completed
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method